spec/views/alchemy/ingredients/richtext_editor_spec.rb

```diff
@@ -32,7 +32,7 @@
     let(:settings) { {tinymce: {plugin: "link"}} }
 
     it "renders a custom configuration" do
-      expect(rendered).to have_selector(".ingredient-editor.richtext script")
+      expect(rendered).to have_selector("textarea[is=alchemy-tinymce][plugin]")
```

Comment:
Maybe add a spec for the `dasherize` as well as `to_json` by using the `tinymce_custom` element?